admin/js/boldgrid-backup-admin-home.js

```diff
@@ -92,7 +92,11 @@ BOLDGRID.BACKUP.HOME = function( $ ) {
 		var $badExtension = $( '#bad_extension' ),
 			$fileSizeWarning = $( '[data-id="upload-backup"]:not(span)' ),
 			$fileTooLarge = $( '#file_too_large' ),
-			$submit = $( $( this ).closest( 'input:submit' ) ),
+			$submit = $(
```

Comment:
Does everything need to be wrapped in the `$(` ?

Reply:
Yeah. That's how it was before I changed it. The only change I made was using .parent().find() rather than .closest() because .closest() wasn't giving us the right result.